manager/assets/modext/widgets/security/modx.panel.access.policy.js

```diff
@@ -63,8 +59,8 @@ MODx.panel.AccessPolicy = function(config) {
                         ,anchor: '100%'
                         ,listeners: {
                             'keyup': {scope:this,fn:function(f,e) {
-                                    Ext.getCmp('modx-policy-header').getEl().update(_('policy')+': '+f.getValue());
-                                }}
+                                Ext.getCmp('modx-policy-header').getEl().update(Ext.util.Format.htmlEncode(f.getValue()));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(f.getValue()));`